backend/src/apiserver/server/experiment_server_test.go

```diff
@@ -422,3 +428,50 @@ func TestValidateCreateExperimentRequest_Multiuser(t *testing.T) {
 		}
 	}
 }
+
+func TestArchiveAndUnarchiveExperiment(t *testing.T) {
+	// Create experiment and runs under it.
+	clients, manager, experiment := initWithExperimentAndPipelineVersion(t)
+	defer clients.Close()
+	runServer := NewRunServer(manager)
+	run1 := &api.Run{
+		Name:               "123",
```

Comment:
Nit: consider using a different name such as "run1" or else?

context: I saw that the test code sometimes use "123" as name and sometimes use it as ID. And I recall one time I was debugging the test to see where a value of "123" came from, it was kinda painful to search the string "123" in our code base. :)

Reply:
@chensun, sure. And as a personal experience, search in VS code is much better than that in github.
@Bobgy agree. And moreover, in general, it is recommended to keep everything simple in tests. E.g, avoid indirections, loops, etc. Here are some internal code style guide regarding tests. 
go/unit-test-practices#control-structures
go/unit-test-practices#dos-and-donts-2
https://g3doc.corp.google.com/devtools/library_club/g3doc/totw/97.md?cl=head